crates/cxx-qt-gen/src/syntax/types.rs

```diff
@@ -0,0 +1,179 @@
+// SPDX-FileCopyrightText: 2023 Klarälvdalens Datakonsult AB, a KDAB Group company <info@kdab.com>
+// SPDX-FileContributor: Leon Matthes <leon.matthes@kdab.com>
+
+// SPDX-License-Identifier: MIT OR Apache-2.0
+
+use crate::syntax::path::*;
```

Comment:
is there any advantage / disadvantage to using `*` vs more explicit includes like we are doing elsewhere?

Reply:
This is a habit I picked up at some point from a personal project.
Generally it means you're avoiding a lot of the bloat in "use" declarations.
They're kinda boilerplate for the most part.
So I like using wildcards to not have to list 20 different things I'm pulling from the same lib.

That being said, effective Rust recommends not using them for compatibility reasons: https://www.lurklurk.org/effective-rust/wildcard.html
So I'll switch that over to explicit imports.

Reply:
Done